processor/src/decoder/mod.rs

```diff
@@ -14,7 +14,8 @@ use vm_core::{
 };
 
 mod trace;
-use trace::DecoderTrace;
+#[allow(clippy::useless_attribute, unused_imports)]
+use trace::{DecoderTrace, TOTAL_USER_OP_HELPERS};
```

Comment:
I think it is ok to import the constant on a separate line with the right attribute.